src/MessagePack.Internal/MessagePack.Internal.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
How important is it that we target `netcoreapp3.0` here? Requiring that machines install prerelease SDKs to be able to build and develop in this repo is an unfortunate requirement I'd rather avoid.

Reply:
We could start with ns2.1 \ ns2.2 until the SDK goes public. Is that more reasonable?

Reply:
Netstandard2.1 isn't released either, is it? Any reason to not just go with a target framework that messagepack already targets?

Reply:
Sorry, I meant netcoreapp2.1 \ netcoreapp2.2

Reply:
Sounds good.